js/index.js

```diff
@@ -1 +1,95 @@
 // Your code goes here
+
+
+// Mouseover & Mouseout - makes nav bar background pink when hovering off and white when hovering over
+const navBar = document.querySelector('.nav');
+navBar.addEventListener('mouseover', (e) => {
+    navBar.style.backgroundColor = 'white';
+});
+
+navBar.addEventListener('mouseout', (e) => {
+    navBar.style.backgroundColor = 'hotpink';
+  });
+
+navBar.addEventListener('click', (e) => {
+    console.log(`${e} has been clicked!`);
+    e.preventDefault();
+    e.stopPropagation();
+})
```

Comment:
Good re-usage of the `navBar` variable, instead of creating a new variable for each event 👍